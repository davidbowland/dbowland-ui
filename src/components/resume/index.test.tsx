import React from 'react'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import Resume from './index'

describe('Resume component', () => {
  test('Ensure snapshot match', () => {
    const { container } = render(<Resume />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
