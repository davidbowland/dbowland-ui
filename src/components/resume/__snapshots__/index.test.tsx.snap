// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component Ensure snapshot match 1`] = `
<div>
  <div
    class="sc-jSFjdj jLwqAM"
  >
    <div
      class="sc-hBMUJo furMN"
    >
      <div
        class="sc-iCoGMd fKsFer"
      />
      <div
        class="sc-crzoAE ypOre"
      >
        <h1
          class="sc-dIsUp iagKpK"
        >
          David Bowland
        </h1>
        <h2
          class="sc-bqGGPW bmUXlr"
        >
          Software Developer
        </h2>
        <h4
          class="sc-gKAaRy tirQr"
        >
          <a
            class="sc-iqAclL fmSdzt"
            href="test-file-stub"
          >
            Download Resume
          </a>
        </h4>
      </div>
      <div
        class="sc-dlnjwi dgDwcj"
      >
        <ul
          class="sc-hKFxyN hXErUM"
        >
          <li
            class="sc-eCApnc eLctso"
          >
            e: 
            <a
              class="sc-bdnxRM kwVBIQ"
              href="mailto:david@dbowland.com"
            >
              david@dbowland.com
            </a>
          </li>
          <li
            class="sc-eCApnc eLctso"
          >
            w: 
            <a
              class="sc-bdnxRM kwVBIQ"
              href="https://dbowland.com"
            >
              dbowland.com
            </a>
          </li>
          <li
            class="sc-eCApnc eLctso"
          >
            m: 
            <a
              class="sc-bdnxRM kwVBIQ"
              href="tel:+14178940079"
            >
              417.894.0079
            </a>
          </li>
        </ul>
      </div>
      <div
        class="sc-dlMDgC jiNouc"
      />
    </div>
    <div
      class="sc-gtsrHT cmmSYJ"
    >
      <section
        class="sc-fnVZcZ jbvnac"
      >
        <div
          class="sc-iemWCZ kRzKvn"
        >
          <h1
            class="sc-dIvrsQ fHCnXW"
          >
            Personal Profile
          </h1>
        </div>
        <div
          class="sc-fFSPTT cazmuJ"
        >
          <article>
            <ul
              class="sc-fujyAs lgroYb"
            >
              <li
                class="sc-pNWdM hkOrul"
              >
                Developer of robust solutions
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Effective translator between geek and English
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Connoisseur of groan-inducing dad jokes
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
        </div>
        <div
          class="sc-dlMDgC jiNouc"
        />
      </section>
      <section
        class="sc-fnVZcZ jbvnac"
      >
        <div
          class="sc-iemWCZ kRzKvn"
        >
          <h1
            class="sc-dIvrsQ fHCnXW"
          >
            Experience
          </h1>
        </div>
        <div
          class="sc-fFSPTT cazmuJ"
        >
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              Technical Lead at
               
              <a
                class="sc-bdnxRM kwVBIQ"
                href="https://www.talentreef.com/"
                rel="noopener noreferrer"
              >
                TalentReef
              </a>
            </h2>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              March 2021 - Present
            </p>
            <ul
              class="sc-fujyAs lgroYb"
            >
              <li
                class="sc-pNWdM hkOrul"
              >
                Assist in migrating existing projects to AWS
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Improve test coverage and automate testing in Node.js
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              Software Developer at
               
              <a
                class="sc-bdnxRM kwVBIQ"
                href="https://www.carfax.com/"
                rel="noopener noreferrer"
              >
                Carfax
              </a>
            </h2>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              March 2020 - March 20201
            </p>
            <ul
              class="sc-fujyAs lgroYb"
            >
              <li
                class="sc-pNWdM hkOrul"
              >
                Develop Spring Boot / Spring Batch apps in Groovy and Java 8
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Develop Node.js apps, using React with webpack for frontend
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Use Agile methodology with two-week sprints organized on Jira
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Deploy apps using Jenkins, either Docker images to Kubernetes on AWS or RPMs to DC 3.0 on-premise
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Went from new hire to acting team leader in less than 10 months
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              Senior Programmer Analyst at
               
              <a
                class="sc-bdnxRM kwVBIQ"
                href="https://www.showmeboone.com/"
                rel="noopener noreferrer"
              >
                Boone County Government
              </a>
            </h2>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              November 2014 - March 2020
            </p>
            <ul
              class="sc-fujyAs lgroYb"
            >
              <li
                class="sc-pNWdM hkOrul"
              >
                Developed full-stack web applications using CSS3, HTML5, and vanilla JavaScript with ASP, ASP.NET, or Java backend
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Developed COBOL and IBM CL applications on IBM System i midrange
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Acted as project manager for two major projects
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Earned three performance-based increases outside annual review
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
        </div>
        <div
          class="sc-dlMDgC jiNouc"
        />
      </section>
      <section
        class="sc-fnVZcZ jbvnac"
      >
        <div
          class="sc-iemWCZ kRzKvn"
        >
          <h1
            class="sc-dIvrsQ fHCnXW"
          >
            Skills
          </h1>
        </div>
        <div
          class="sc-fFSPTT cazmuJ"
        >
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              Languages
            </h2>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              Proficient in
            </p>
            <ul
              class="sc-jrsJWt fLIvTG"
            >
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Groovy
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Java
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                JavaScript
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Python
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                TypeScript
              </li>
            </ul>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              Familiar with
            </p>
            <ul
              class="sc-jrsJWt fLIvTG"
            >
              <li
                class="sc-kEqXSa iqTgBr"
              >
                ASP/ASP.NET
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                C++
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                COBOL
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                PHP
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Terraform
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              SQL
            </h2>
            <ul
              class="sc-jrsJWt fLIvTG"
            >
              <li
                class="sc-kEqXSa iqTgBr"
              >
                DB2
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Microsoft SQL Server (T/SQL)
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                MySQL
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Oracle Database
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                PostgreSQL
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              NoSQL
            </h2>
            <ul
              class="sc-jrsJWt fLIvTG"
            >
              <li
                class="sc-kEqXSa iqTgBr"
              >
                DynamoDB
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              Technologies
            </h2>
            <ul
              class="sc-jrsJWt fLIvTG"
            >
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Argo
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Docker
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Git
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Kibana
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Kubernetes
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Jenkins
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Jira
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                NewRelic
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              AWS
            </h2>
            <ul
              class="sc-jrsJWt fLIvTG"
            >
              <li
                class="sc-kEqXSa iqTgBr"
              >
                CloudWatch
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                DynamoDB
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Lambda
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                RDS
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                S3
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Systems Manager
              </li>
              <li
                class="sc-kEqXSa iqTgBr"
              >
                Secrets Manager
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
        </div>
        <div
          class="sc-dlMDgC jiNouc"
        />
      </section>
      <section
        class="sc-fnVZcZ jbvnac"
      >
        <div
          class="sc-iemWCZ kRzKvn"
        >
          <h1
            class="sc-dIvrsQ fHCnXW"
          >
            Education
          </h1>
        </div>
        <div
          class="sc-fFSPTT cazmuJ"
        >
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              <a
                class="sc-bdnxRM kwVBIQ"
                href="https://ccis.edu/"
                rel="noopener noreferrer"
              >
                Columbia College — Columbia, MO
              </a>
            </h2>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              Bachelor of Science — GPA 3.68
            </p>
            <p
              class="sc-ksluID sc-hHEiqL gAVCtE bwKQMd"
            >
              Major: Computer Science — Minor: Business
            </p>
            <ul
              class="sc-fujyAs lgroYb"
            >
              <li
                class="sc-pNWdM hkOrul"
              >
                Graduated cum laude with a GPA of 3.68/4.0
              </li>
              <li
                class="sc-pNWdM hkOrul"
              >
                Earned an A in all computer science classes
              </li>
            </ul>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
          <article>
            <h2
              class="sc-bkbkJK MFdWM"
            >
              <a
                class="sc-bdnxRM kwVBIQ"
                href="https://www.cpsk12.org/HHS"
                rel="noopener noreferrer"
              >
                David H. Hickman High School — Columbia, MO
              </a>
            </h2>
            <div
              class="sc-dlMDgC jiNouc"
            />
          </article>
        </div>
        <div
          class="sc-dlMDgC jiNouc"
        />
      </section>
    </div>
  </div>
</div>
`;
